### PR TITLE
Add labels key to i18n/ja/OWNERS

### PR DESCRIPTION
**Pull request by atoato88**
This PR adds `labels` item to `i18n/ja/OWNERS` for automatic `language/ja` labeling.

### CODE REVIEW

File: i18n/ja/OWNERS [hunk 1]
```diff
@@ -2,3 +2,6 @@ approvers:
   - shu-mutou
   - atoato88
 
+labels:
+- language/ja
```
Comment (floreks) [inline i18n/ja/OWNERS]: Can you add proper labels to OWNERS files in other i18n directories?

Comment (atoato88) [reply i18n/ja/OWNERS]: Thank you for comment.
I will do it, after confirm that "language/ja" is attached automatically by other PR.

**Pull request merged**